lib/public/views/Logs/Create/TemplatedLogCreationModel.js

```diff
@@ -49,10 +50,14 @@ export class TemplatedLogCreationModel extends LogCreationModel {
      *
      * @param {function} [onCreation] function called when log is created, with the id of the created log
      * @param {LogCreationRelations} relations the relations of the log
+     * @param {LogCreationTemplateData} templateData the template data of the log
      */
-    constructor(onCreation, relations) {
+    constructor(onCreation, relations, templateData) {
         super(onCreation, relations);
 
+        this._relations = relations;
+        this._templateData = templateData;
```

Comment:
Yo do not need to store this as property, you should simply pass it as parameter to `this.useTemplate`

Reply:
Do you? I do not see it used

Reply:
<img width="882" alt="Screenshot 2024-12-11 at 11 33 30" src="https://github.com/user-attachments/assets/9f67a184-acb2-40d8-822a-a1d870c72f53">
Here the useTemplate should get called and this method doesnt have access to the templateData

Reply:
That makes sense, I fixed it